scipy/optimize/_zeros_py.py

```diff
@@ -1393,3 +1394,222 @@ def toms748(f, a, b, args=(), k=1,
                           maxiter=maxiter, disp=disp)
     x, function_calls, iterations, flag = result
     return _results_select(full_output, (x, function_calls, iterations, flag))
+
+
+def chandrupatla(f, x0, x1, args=(), xtol=_xtol,
+                 rtol=_rtol, maxiter=_iter, callback=None):
+
+    """
+    As written in [Sachs2015]_ which in turn is based on Chandrupatla's
+    algorithm as described in [Scherer2018]_.
```

Comment:
Instead of resolving before changing, there are two things you can do instead:

- You can go to Files Changed, "Add suggestion to batch" for all the suggestions you agree with, and commit the batch. That will automatically resolve the comment, but that's OK because by committing the suggestion, I can see that my comment has been addressed..
- If you prefer to make the changes manually, you could add an emoji to keep track of the suggestions you think you've addressed. I'll click the resolve button after I have verified that they are resolved.

Reply:
Ah okay! Sorry about that I didn't realize the resolve button was something for you to keep track of. I'll use the second option in the future.